src/solver/cmake/variable.cmake

```diff
@@ -129,6 +129,7 @@ set(SRC_VARIABLE_ECONOMY
 		variable/economy/waterValue.h
 		variable/economy/hydroCost.h
 		variable/economy/unsupliedEnergy.h
+		variable/economy/domesticUnsupliedEnergy.h

```

Comment:
switch domesticUnsupliedEnergy for domesticUnsuppliedEnergy, and the same for the other places